client/stylesheets/index.css

```diff
@@ -20,5 +20,5 @@
 }
 
 .jolly-roger {
-    font-family: "Helvetica Neue", "Helvetica", "Arial", sans-serif, "Noto Color Emoji", "Apple Color Emoji", "Segoe UI Emoji";
+    font-family: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Arial", sans-serif, "Noto Color Emoji", "Apple Color Emoji", "Segoe UI Emoji";
```

Comment:
This is a good catch. Popovers also set their font to Bootstrap's default directly in their style property. Rather than try to patch these sorts of cases, I think it makes more sense to set up a Bootstrap theme and set the font variables. (A quick review reveals a lot of other scss that's just overriding Bootstrap and could easily be moved into a theme. That seems out of scope for this PR, but I've added it to my list.)

Unfortunately, the `@use` rule isn't available because fourseven:scss uses node-sass, which was deprecated before it was added. In the future, this can be factored slightly differently to take advantage of `@use`.